content/docs/release-notes/archive/21-11-release-notes.md

```diff
@@ -0,0 +1,339 @@
+
+Most importantly, Prefabs are now represented as a single object in the world, and you must use the **Edit Prefab** action on a Prefab in the Editor in order to edit the prefab itself. Prefab editing is available through **double-clicking** on an object in the Editor. While locked to the prefab edit mode, other instances of the prefab visible through the Editor viewport will receive real-time updates.
+
+And another change that we thought was so important that it deserves its own callout: This means is no more "sticky-select" in the Editor by default! You no longer have to double-click when deselecting an entity that you're editing. If you'd like to re-enable this feature, it can be changed in Editor Settings or with the `/Amazon/Preferences/Editor/StickySelect` registry setting.
+
+### Heightfield-based terrain
+
+We now have an experimental terrain system! Even basic terrain is important for designing outdoor environments and larger-scale worlds. The terrain system uses [heightmaps](https://en.wikipedia.org/wiki/Heightmap) to define deformations over a geometric plane to create world terrain. Anyone familiar with heightmaps should be able to play around with terrain now, and we encourage all of our users to try this feature when creating outdoor scenes in Open 3D Engine.
+
+![Heightfield Terrain editing in the O3DE Editor](/images/release-notes/2111-1/terrain.png)
+
+For the full information on implementation progress and the planned design, please see [O3DE SIG-Content RFC #4](https://github.com/o3de/sig-content/blob/main/rfcs/rfc-4-terrain-system.md).
+
+### Project manager external Gems support
+
+Open 3D Engine Project Manager how has improvements which allow the support of Gems which exist outside of the official Gem catalog! This feature comes along with the ability to register a source control repository with the Project Manager, and use this as an external source to pull and install Gems for your project. In addition, we've fixed a critical bug: Project Manager now correctly auto-adds and auto-displays Gem dependencies which affect your project configuration.
+
+### Script canvas performance and editing
+
+2111.1 contains plenty of performance improvements, but one which affects all project runtimes (and working in the Editor!) is worth calling out. Script Canvas now uses JSON as its new file format, which gives an average 71% size file reduction. Smaller files means faster loads, too! And not just that, but with the migration away from XML serialization and towards more flexible JSON tools, it's easier than ever to write external tools and scripts that can ingest Script Canvas JSON files.
```

Comment:
```suggestion
2111.1 contains plenty of performance improvements, but one which affects all project runtimes and performance in the Editor is worth calling out. Script Canvas now uses JSON as its file format instead of XML, which gives an average 71% size file reduction. Smaller files means faster loads, too! And not just that, but with the migration away from XML serialization and towards more flexible JSON tools, it's easier than ever to write external tools and scripts that can ingest Script Canvas JSON files.
```